ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -126,55 +126,120 @@ struct FluentButtonBody: View {
-                        radius: shouldUsePressedShadow ? tokens.pressedShadow1Blur : tokens.restShadow1Blur,
-                        x: shouldUsePressedShadow ? tokens.pressedShadow1DepthX : tokens.restShadow1DepthX,
-                        y: shouldUsePressedShadow ? tokens.pressedShadow1DepthY : tokens.restShadow1DepthY)
-                .shadow(color: shouldUsePressedShadow ? tokens.pressedShadow2Color : tokens.restShadow2Color,
-                        radius: shouldUsePressedShadow ? tokens.pressedShadow2Blur : tokens.restShadow2Blur,
-                        x: shouldUsePressedShadow ? tokens.pressedShadow2DepthX : tokens.restShadow2DepthX,
-                        y: shouldUsePressedShadow ? tokens.pressedShadow2DepthY : tokens.restShadow2DepthY)
-        })
+
+        @ViewBuilder
+        var buttonBackground: some View {
+            if tokens.borderSize > 0 {
+                buttonContent.background(
+                    AnyView(RoundedRectangle(cornerRadius: tokens.borderRadius)
+                                .strokeBorder(lineWidth: tokens.borderSize, antialiased: false)
+                                .foregroundColor(Color(borderColor))
+                                .contentShape(Rectangle())))
+            } else {
+                buttonContent.background(
+                    AnyView(RoundedRectangle(cornerRadius: tokens.borderRadius)
+                                .fill(Color(backgroundColor))))
+            }
+        }
+
+        let shadow1Color: Color
+        let shadow1Blur: CGFloat
+        let shadow1DepthX: CGFloat
+        let shadow1DepthY: CGFloat
+        let shadow2Color: Color
+        let shadow2Blur: CGFloat
+        let shadow2DepthX: CGFloat
+        let shadow2DepthY: CGFloat
+        if shouldUsePressedShadow {
+            shadow1Color = tokens.pressedShadow1Color
+            shadow1Blur = tokens.pressedShadow1Blur
+            shadow1DepthX = tokens.pressedShadow1DepthX
+            shadow1DepthY = tokens.pressedShadow1DepthY
+            shadow2Color = tokens.pressedShadow2Color
+            shadow2Blur = tokens.pressedShadow2Blur
+            shadow2DepthX = tokens.pressedShadow2DepthX
+            shadow2DepthY = tokens.pressedShadow2DepthY
+        } else {
+            shadow1Color = tokens.restShadow1Color
+            shadow1Blur = tokens.restShadow1Blur
+            shadow1DepthX = tokens.restShadow1DepthX
+            shadow1DepthY = tokens.restShadow1DepthY
+            shadow2Color = tokens.restShadow2Color
+            shadow2Blur = tokens.restShadow2Blur
+            shadow2DepthX = tokens.restShadow2DepthX
+            shadow2DepthY = tokens.restShadow2DepthY
+        }
+
+        @ViewBuilder
+        var button: some View {
+            if isFloatingStyle {
+                buttonBackground
+                    .clipShape(Capsule())
+                    .shadow(color: shadow1Color,
+                           radius: shadow1Blur,
```

Comment:
Note: we don't align colons (`:`) in Swift like we do in Objective-C. The aforementioned `^I` will left-align these as is the Swift standard.